charts/stackrox/templates/configure-stackrox-job.yaml

```diff
@@ -19,9 +19,12 @@ spec:
             - |
               test {{ .Values.verbose }} == true && set -x
 
-              # wait for operator
+              # wait for CRD
               while [ true ]; do oc get crd centrals.platform.stackrox.io; if [ $? -eq 0 ]; then break; fi ; sleep 5s; done
 
+              # wait for operator to spin-up
+              while test True != $(oc -n openshift-operators get pod -l app=rhacs-operator -o jsonpath='{.status.conditions[?(@.type == "Ready")].status}'); do sleep 5; done
+
```

Comment:
@jfilipcz do we need a timeout for this loop? Seems a bit happy path to assume that the pod(s) will always end up being Ready.

Reply:
Yup, agreed - I was relying on helm install timeout, but including the timeout in the job itself seems to be a better idea